dist/autocomplete.angular.js

```diff
@@ -1353,12 +1353,40 @@
 	// shim for using process in browser
 
 	var process = module.exports = {};
+
+	// cached from whatever global is present so that test runners that stub it
```

Comment:
Added code is just an update on https://github.com/defunctzombie/node-process/pull/61/files